src/components/v2/V2Project/V2PayButton/V2ConfirmPayModal/index.tsx

```diff
@@ -110,7 +111,7 @@ export function V2ConfirmPayModal({
   )
   let nftRewardTier: NftRewardTier | null = null
 
-  if (nftRewardTiers && featureFlagEnabled('nftRewards')) {
+  if (nftRewardTiers && featureFlagEnabled(FEATURE_FLAGS.NFT_REWARDS)) {
```

Comment:
Introduce single source of truth for feature flag names

Reply:
Sweet